### ISSUE TITLE
Correct and not correct chain of queries only shows the latest query as correct

### ISSUE THREAD

**Issue by Acefall** [data-hydenv/hydenv-database#8]
Not a fatal error but still confusing.
Correct and not correct chain of queries in the "Task Description" tab only shows the latest query as correct even tho the previous queries also were correct.

Reproduce:
Enter correct query. Hit "Execute" button multiple times.

**Comment by mmaelicke** [comment]
Yeah, I know about that. Unfortunately, it is only possible to fix this behavior with substatial efforts. The problem is that, while each unsuccessful attempt is stored, the successful attempt is always overwritten. That means, older successful attempts 'drop' out of the chain. This is due to a misdesign of the data model, because I assumed that, whenever an exercise is solved, the student would click on 'next' and that's it.

I'll keep this issue open to not forget about it, but it is of very low priority due to the cost-benefit mismatch...
